curriculum/challenges/english/25-front-end-development/lab-pyramid-generator/66f2836c459cfb16ae76f24f.md

```diff
@@ -46,16 +46,16 @@ Your `pyramid` function should have three parameters.
 assert.lengthOf(pyramid, 3);
 ```
 
-`pyramid('o', 4, false)` should return `"\n   o\n  ooo\n ooooo\nooooooo\n"`.
+`pyramid("o", 4, false)` should return `"\n   o\n  ooo\n ooooo\nooooooo\n"`.
 
 ```js
-assert.equal(pyramid('o', 4, false), "\n   o\n  ooo\n ooooo\nooooooo\n")
+assert.equal(pyramid("o", 4, false), "\n   o\n  ooo\n ooooo\nooooooo\n")
```

Comment:
the code in the tests is not user-facing so it is not necessary to change fyi (not that it's wrong)